base/applications/network/netstat/netstat.c

```diff
@@ -103,6 +100,8 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                         break;
                     case L'p':
                         bDoShowProtoCons = TRUE;
+                        if(i+1 >= argc)
```

Comment:
```suggestion
                        if (i + 1 >= argc)
```

Reply:
Lol, I already anticipated that and just fixed it.